src/components/Inquiry/Answer.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+import { useSelector } from 'react-redux'
+import CustomIconButton from '../custom/CustomIconButton'
+import store from '../../state/store/configureStore'
+import { useTranslation } from 'react-i18next'
+import CustomRadioButton from '../../components/custom/CustomRadioButton'
+
+const Answer = ({ text, type, placeholder, questionKey }) => {
+  const [inputValue, setInputValue] = useState(
+    store.getState().formData[questionKey]
+  )
+  const { t } = useTranslation()
+  const filled = useSelector((state) => state.filledAnswers[questionKey])
+
+  const setAnswer = (event) => {
+    event.preventDefault()
+    if (filled) {
+      store.dispatch({
+        type: 'SWITCH_ANSWER_FILLED_STATUS',
+        payload: { key: questionKey, filled: false },
+      })
+    } else {
+      store.dispatch({
+        type: 'SWITCH_ANSWER_FILLED_STATUS',
+        payload: { key: questionKey, filled: true },
+      })
+      store.dispatch({
+        type: 'SET_ANSWERS',
+        payload: { key: questionKey, answer: inputValue },
+      })
```

Comment:
Small refactoring, but think you can wrap the objects in an array
`store.dispatch([{type: "bll2"}, {type: "BKOWFJ"}])`

Reply:
smart, we'll do it